service/pom.xml

```diff
@@ -18,5 +18,10 @@
       <artifactId>persistence</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+        <groupId>com.google.protobuf</groupId>
+        <artifactId>protobuf-java-util</artifactId>
+        <version>4.26.1</version>
```

Comment:
Set this version to match that of the `protobuf-java` in `smile-commons` (see [here](https://github.com/mskcc/smile-commons/blob/master/pom.xml#L112)) for compatibility.